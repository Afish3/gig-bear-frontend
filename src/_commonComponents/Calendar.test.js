import React from 'react';
import { render, screen } from '@testing-library/react';
import GigsCalendar from './Calendar';

describe('GigsCalendar Interaction', () => {
  it('updates availability on calendar interaction', async () => {
    // const mockEditProfile = jest.spyOn(API, 'editProfile').mockResolvedValue({});

    render(<GigsCalendar username="testUser" availability={[]} />);

    // // Assert that the expected messages are present on the screen
    expect(screen.getByText('Interact with the calendar')).toBeInTheDocument();
    expect(screen.getByText('to update availabile dates and times.')).toBeInTheDocument();
    expect(screen.getByText('Click to update when you are finished!')).toBeInTheDocument();
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<GigsCalendar username="testUser" availability={[]} />);
    expect(asFragment).toMatchSnapshot();
  });
});