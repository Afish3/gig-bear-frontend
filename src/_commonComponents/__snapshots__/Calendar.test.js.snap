// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GigsCalendar Interaction matches snapshot 1`] = `[Function]`;
